Cert creating is broken

```
Certificate request self-signature ok                            
subject=O = NSPCC                                                
⇒ Creating TLS certs to NeoFS node                               
                                                                 
Error checking x509 extension section san                        
make: *** [services/storage/artifacts.mk:8: get.storage] Error 1 
```

--- a/services/storage/artifacts.mk
+++ b/services/storage/artifacts.mk
@@ -2,7 +2,7 @@
 
 CURRENT_DIR=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 STORAGE_DIR=$(patsubst %/,%,$(CURRENT_DIR))
-SSL_CONFIG=$(shell mktemp)
+SSL_CONFIG:=$(shell mktemp)
 
 get.storage:
 	@echo "⇒ Creating TLS certs to NeoFS node"

